third_party/airflow/pyproject.toml

```diff
@@ -24,5 +24,8 @@ test = ["pytest==7.3.1", "coverage>=6.5.0", "pytest-asyncio==0.21.0"]
 docs = ["sphinx", "sphinx-jekyll-builder"]
 
 [build-system]
-requires = ["setuptools"]
-build-backend = "setuptools.build_meta"
+requires = ["flit_core>=3.8.0,<4"]
```

Comment:
What is `flit_core`?

Reply:
https://github.com/pypa/flit

I couldn't get `pip install -e` to work with `armada-airflow` on setuptools so I tried another build backend.